lib/tty/command/printers/pretty_on_error.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'pastel'
+
+require_relative 'pretty'
+
+module TTY
+  class Command
+    module Printers
+      class PrettyOnError < Pretty
+        def print_command_start(cmd, *args)
+          super(cmd, *args)
+
+          @out_data = ''
+          @err_data = ''
+        end
+
+        def print_command_out_data(cmd, *args)
+          message = args.map(&:chomp).join(' ')
+
+          @out_data += write("\t#{message}", cmd.uuid, true)
```

Comment:
Don't know if I can replace these with a `super` call, so the line `20` is actually duplicated.